libraries/AP_HAL_ChibiOS/hwdef/CubeOrange-Ottano/scripts/ICE-auto-warmup.lua

```diff
@@ -0,0 +1,153 @@
+-- This Script runs through an warmup up sequence for the ICE engine. 
+-- Once the engine is started it will use the Idle governor to modify the engine RPM to desired value until CHT's reach desired value.
+-- Once the CHT's reach desired value the script will switch to the normal governor and set the desired RPM to the normal value.
+
+local UPDATE_HZ = 1
+
+local MAV_SEVERITY_INFO = 6
+local MAV_SEVERITY_NOTICE = 5
+local MAV_SEVERITY_WARNING = 4
+local MAV_SEVERITY_ERROR = 3
+local MAV_SEVERITY_CRITICAL = 2
+local MAV_SEVERITY_ALERT = 1
+local MAV_SEVERITY_EMERGENCY = 0
+
+gcs:send_text(MAV_SEVERITY_INFO, "ICE Warmup: Loaded")
+
+--Track State of Warmup as to not print too many messages
+local STATUS_NOT_DONE = 0
+local STATUS_IGNITION = 1
+local STATUS_IDLE = 2
+local STATUS_STEP = 3
+local STATUS_DONE = 4
+
+local Warmup_Status = STATUS_NOT_DONE
+
+-- Bind Param Utilities
+local PARAM_TABLE_KEY = 69
+local PARAM_TABLE_PREFIX = "WARMUP_"
+local function bind_param(name)
+    local p = Parameter()
+    assert(p:init(name), string.format('could not find %s parameter', name))
+    return p
+end
+
+function bind_add_param(name, idx, default_value)
+   assert(param:add_param(PARAM_TABLE_KEY, idx, name, default_value), string.format('could not add param %s', name))
+   return Parameter(PARAM_TABLE_PREFIX .. name)
+end
+
+--Setup Warmup Parameters
+assert(param:add_table(PARAM_TABLE_KEY, PARAM_TABLE_PREFIX, 13), 'could not add param table')
```

Comment:
```suggestion
assert(param:add_table(PARAM_TABLE_KEY, PARAM_TABLE_PREFIX, 13), 'could not add param table')
```
assert(param:add_table(PARAM_TABLE_KEY, PARAM_TABLE_PREFIX, 5), 'could not add param table')

Reply:
We have 4 PARAMETERS, don't need 13 slots